.gitignore

```diff
@@ -15,6 +15,8 @@ rerun.txt
 pickle-email-*.html
 .idea
 coverage
+Gemfile.lock
```

Comment:
do not add gemfile.lock to gitignore. this is important file, it has information about installed ruby gems versions!!!